src/services/ServiceManager.ts

```diff
@@ -35,10 +35,11 @@ export type BackgroundTask =
   | { name: 'SELF_HOST_BACKUP'; data: SelfHostData }
   | { name: 'SELF_HOST_RESTORE'; data: SelfHostData }
   | { name: 'MIGRATE_NOVEL'; data: MigrateNovelData }
-  | {
-      name: 'DOWNLOAD_CHAPTER';
-      data: { chapterId: number; novelName: string; chapterName: string };
-    };
+  | DownloadChapterTask;
+export type DownloadChapterTask = {
+  name: 'DOWNLOAD_CHAPTER';
+  data: { chapterId: number; novelName: string; chapterName: string };
+};
```

Comment:
This change should have been part of the first commit of the 2 but idk how to split single files up with command line and itll prob get squashed together anyway :shrug: